cmd/tdex/feeder.go

```diff
@@ -0,0 +1,305 @@
+			},
+			&cli.StringFlag{
+				Name:  "quote_asset",
+				Usage: "quote asset of the market for which the price feed is created",
+			},
+			&cli.StringFlag{
+				Name:  "source",
+				Usage: "price source to use, e.g. kraken, bitfinex, coinbase etc",
+			},
+			&cli.StringFlag{
+				Name:  "ticker",
+				Usage: "ticker of the market, e.g. XBT/USDT, XBT/EUR etc",
+			},
+		},
+	}
+	startPriceFeed = &cli.Command{
+		Name:   "start",
+		Usage:  "starts price feed",
+		Action: startPriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "id",
+				Usage: "id of the price feed to start",
+			},
+		},
+	}
+	stopPriceFeed = &cli.Command{
+		Name:   "stop",
+		Usage:  "stops price feed",
+		Action: stopPriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "id",
+				Usage: "id of the price feed to start",
+			},
+		},
+	}
+	updatePriceFeed = &cli.Command{
+		Name:   "update",
+		Usage:  "update a price feed source and ticker",
+		Action: updatePriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "id",
+				Usage: "id of the price feed to start",
+			},
+			&cli.StringFlag{
+				Name:  "source",
+				Usage: "price source to use, e.g. kraken, bitfinex, coinbase etc",
+			},
+			&cli.StringFlag{
+				Name:  "ticker",
+				Usage: "ticker of the market, e.g. XBT/USDT, XBT/EUR etc",
+			},
+		},
+	}
+	removePriceFeed = &cli.Command{
+		Name:   "remove",
+		Usage:  "remove a price feed",
+		Action: removePriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "id",
+				Usage: "id of the price feed to start",
+			},
+		},
+	}
+	getPriceFeed = &cli.Command{
+		Name:   "get",
+		Usage:  "get a price feed",
+		Action: getPriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "base_asset",
+				Usage: "base asset of the market for which the price feed is created",
+			},
+			&cli.StringFlag{
+				Name:  "quote_asset",
+				Usage: "quote asset of the market for which the price feed is created",
+			},
+		},
+	}
+	listPriceFeeds = &cli.Command{
+		Name:   "list",
+		Usage:  "list price feeds",
+		Action: listPriceFeedsAction,
+	}
+	listSources = &cli.Command{
+		Name:   "sources",
+		Usage:  "list price feed sources",
+		Action: listSourcesAction,
+	}
+)
+
+func addPriceFeedAction(ctx *cli.Context) error {
+	client, cleanup, err := getFeederClient(ctx)
+	if err != nil {
+		return err
+	}
+	defer cleanup()
+
+	baseAsset := ctx.String("base_asset")
+	quoteAsset := ctx.String("quote_asset")
+	source := ctx.String("source")
+	ticker := ctx.String("ticker")
+
+	if baseAsset == "" || quoteAsset == "" || source == "" || ticker == "" {
+		return cli.Exit("base_asset, quote_asset, source and ticker are required", 1)
+	}
+
+	if _, err := client.AddPriceFeed(ctx.Context, &daemonv2.AddPriceFeedRequest{
+		Market: &tdexv2.Market{
+			BaseAsset:  baseAsset,
+			QuoteAsset: quoteAsset,
+		},
+		Source: source,
+		Ticker: ticker,
+	}); err != nil {
+		return cli.Exit(err, 1)
```

Comment:
never used `cli.Exit(err, 1)` in other commands, let's keep returning an error as always (please make this change wherever required):
```suggestion
		return err
```

Reply:
Done